apps/glusterfs/app.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/boltdb/bolt"
 	"github.com/gorilla/mux"
 	"github.com/heketi/heketi/executors"
+	"github.com/heketi/heketi/executors/caiexec"
```

Comment:
nit: `caiexec` is not intuitive, i don't have a strong preference, but maybe something like `rawexec` sounds better?

You can leave it as it, depending on how much work you need to do find-replace.